histogram/histogram.go

```diff
@@ -18,12 +18,12 @@ type Config struct {
 // See https://godoc.org/github.com/prometheus/client_golang/prometheus#MustNewConstHistogram.
 type Histogram struct {
 	// count is the number of entries added to the Histogram.
-	count uint64
+	count int32
 	// sum is the total sum of all entries added to the Histogram.
 	sum float64
 	// buckets is a map of upper bounds to cumulative counts of entries,
 	// excluding the +Inf bucket.
-	buckets map[float64]uint64
+	buckets map[float64]int32
```

Comment:
Why not `uint32`? That gives two times bigger number.

Reply:
Nice shout